plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -88,15 +81,8 @@ protected QueryRunner createQueryRunner()
                         "delta.register-table-procedure.enabled", "true"),
                 hiveMinioDataLake.getMinio().getMinioAddress(),
                 hiveMinioDataLake.getHiveHadoop());
-        queryRunner.execute("CREATE SCHEMA " + SCHEMA + " WITH (location = 's3://" + bucketName + "/" + SCHEMA + "')");
-        TpchTable.getTables().forEach(table -> {
-            String tableName = table.getTableName();
-            hiveMinioDataLake.copyResources(resourcePath + tableName, SCHEMA + "/" + tableName);
-            queryRunner.execute(format("CALL system.register_table('%1$s', '%2$s', 's3://%3$s/%1$s/%2$s')",
-                    SCHEMA,
-                    tableName,
-                    bucketName));
-        });
+        queryRunner.execute("CREATE SCHEMA " + SCHEMA + " WITH (location = 's3://" + BUCKET_NAME + "/" + SCHEMA + "')");
+        copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, queryRunner.getDefaultSession(), REQUIRED_TPCH_TABLES);
```

Comment:
Adding here the logs of the related operations while running the test on my machine in case anybody is interested:

```
2023-04-21T06:41:49.983+0200	INFO	main	io.trino.testing.QueryAssertions	Imported 1500 rows for customer in 4.10s
2023-04-21T06:41:50.184+0200	INFO	dispatcher-query-5	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044149_00002_m52x4 :: FINISHED :: elapsed 181ms :: planning 80ms :: waiting 12ms :: scheduling 41ms :: running 40ms :: finishing 20ms :: begin 2023-04-21T06:41:49.994+02:00 :: end 2023-04-21T06:41:50.175+02:00
2023-04-21T06:41:50.637+0200	INFO	dispatcher-query-1	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044150_00003_m52x4 :: FINISHED :: elapsed 357ms :: planning 71ms :: waiting 226ms :: scheduling 264ms :: running 12ms :: finishing 10ms :: begin 2023-04-21T06:41:50.273+02:00 :: end 2023-04-21T06:41:50.630+02:00
2023-04-21T06:41:50.639+0200	INFO	main	io.trino.testing.QueryAssertions	Running import for nation
2023-04-21T06:41:51.416+0200	INFO	dispatcher-query-5	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044150_00004_m52x4 :: FINISHED :: elapsed 766ms :: planning 134ms :: waiting 139ms :: scheduling 151ms :: running 44ms :: finishing 437ms :: begin 2023-04-21T06:41:50.645+02:00 :: end 2023-04-21T06:41:51.411+02:00
2023-04-21T06:41:51.419+0200	INFO	main	io.trino.testing.QueryAssertions	Imported 25 rows for nation in 779.30ms
2023-04-21T06:41:51.498+0200	INFO	dispatcher-query-6	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044151_00005_m52x4 :: FINISHED :: elapsed 68ms :: planning 23ms :: waiting 2ms :: scheduling 12ms :: running 22ms :: finishing 11ms :: begin 2023-04-21T06:41:51.423+02:00 :: end 2023-04-21T06:41:51.491+02:00
2023-04-21T06:41:51.776+0200	INFO	dispatcher-query-3	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044151_00006_m52x4 :: FINISHED :: elapsed 268ms :: planning 36ms :: waiting 197ms :: scheduling 216ms :: running 7ms :: finishing 9ms :: begin 2023-04-21T06:41:51.504+02:00 :: end 2023-04-21T06:41:51.772+02:00
2023-04-21T06:41:51.778+0200	INFO	main	io.trino.testing.QueryAssertions	Running import for orders
2023-04-21T06:41:52.775+0200	INFO	dispatcher-query-1	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044151_00007_m52x4 :: FINISHED :: elapsed 987ms :: planning 169ms :: waiting 120ms :: scheduling 137ms :: running 119ms :: finishing 562ms :: begin 2023-04-21T06:41:51.784+02:00 :: end 2023-04-21T06:41:52.771+02:00
2023-04-21T06:41:52.776+0200	INFO	main	io.trino.testing.QueryAssertions	Imported 15000 rows for orders in 997.91ms
2023-04-21T06:41:52.860+0200	INFO	dispatcher-query-6	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044152_00008_m52x4 :: FINISHED :: elapsed 75ms :: planning 22ms :: waiting 2ms :: scheduling 13ms :: running 31ms :: finishing 9ms :: begin 2023-04-21T06:41:52.780+02:00 :: end 2023-04-21T06:41:52.855+02:00
2023-04-21T06:41:53.136+0200	INFO	dispatcher-query-0	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044152_00009_m52x4 :: FINISHED :: elapsed 264ms :: planning 43ms :: waiting 187ms :: scheduling 207ms :: running 6ms :: finishing 8ms :: begin 2023-04-21T06:41:52.867+02:00 :: end 2023-04-21T06:41:53.131+02:00
2023-04-21T06:41:53.137+0200	INFO	main	io.trino.testing.QueryAssertions	Running import for region
2023-04-21T06:41:53.782+0200	INFO	dispatcher-query-0	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044153_00010_m52x4 :: FINISHED :: elapsed 638ms :: planning 137ms :: waiting 121ms :: scheduling 130ms :: running 19ms :: finishing 352ms :: begin 2023-04-21T06:41:53.140+02:00 :: end 2023-04-21T06:41:53.778+02:00
2023-04-21T06:41:53.783+0200	INFO	main	io.trino.testing.QueryAssertions	Imported 5 rows for region in 646.23ms
2023-04-21T06:41:53.840+0200	INFO	dispatcher-query-0	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044153_00011_m52x4 :: FINISHED :: elapsed 50ms :: planning 17ms :: waiting 2ms :: scheduling 9ms :: running 15ms :: finishing 9ms :: begin 2023-04-21T06:41:53.787+02:00 :: end 2023-04-21T06:41:53.837+02:00
2023-04-21T06:41:54.039+0200	INFO	dispatcher-query-5	io.trino.event.QueryMonitor	TIMELINE: Query 20230421_044153_00012_m52x4 :: FINISHED :: elapsed 190ms :: planning 29ms :: waiting 131ms :: scheduling 144ms :: running 9ms :: finishing 8ms :: begin 2023-04-21T06:41:53.845+02:00 :: end 2023-04-21T06:41:54.035+02:00
2023-04-21T06:41:54.040+0200	INFO	main	io.trino.testing.QueryAssertions	Loading from tpch.tiny complete in 8.15s
```